src/index.js

```diff
@@ -0,0 +1,59 @@
+export function CfRepeater(Vue) {
+    return Vue.extend({
+        props: {
+            collection: {
+                required: true
+            },
+            wrap: {
+                type: String,
+                default: 'div'
+            }
+        },
+
+        computed: {
+            /*
+             * Get items from *injected* collection. Reactive.
+             *
+             * @returns {*}
+             */
+            collectionItems () {
```

Comment:
Maybe just `items`, since it's a property, and "the repeater's items" is quite intuitive?